index.html

```diff
@@ -2,192 +2,192 @@
       <section class="celebrating">
-        <h3>Celebrating 15 Years</h3>
-        <p>Straight to the Point Archery is proud to serve Washington State and the greater Puget Sound Area for all your archery lessons and event needs. Including Seattle, Shoreline, Mountlake Terrace, Bellevue and the greater Eastside.</p>
-        <p>Archery Classes • Private Lessons • Recreation League • Birthday Parties and Corporate Parties • Archery Tag</p>
+        <h3>- Celebrating 15 Years -</h3>
+        <p class="c-p1">Straight to the Point Archery is proud to serve Washington State and the greater Puget Sound Area for all your archery lessons and event needs. Including Seattle, Shoreline, Mountlake Terrace, Bellevue and the greater Eastside.</p>
+        <p class="c-p2">Archery Classes • Private Lessons • Recreation League • Birthday Parties and Corporate Parties • Archery Tag</p>
       </section>
       <!-- end celebrating section -->
       <!-- section counter section start -->
-      <div class="counter-row">
-        <div class="counter-column">
-          <div class="card">
-            <p><i class="fas fa-warehouse"></i></p>
-            <h3>28</h3>
-            <p>Indoor Lanes</p>
+      <div class="section-counter">
+        <div class="counter-row">
+          <div class="counter-column">
+            <div class="card">
+              <p><i class="fas fa-warehouse f-card"></i></p>
+              <h3>28</h3>
+              <p class="cc-1">Indoor Lanes</p>
+            </div>
           </div>
-        </div>
-        <div class="counter-column">
-          <div class="card">
-            <p><i class="fas fa-tree"></i></p>
-            <h3>26</h3>
-            <p>Outdoor Lanes</p>
+          <div class="counter-column">
+            <div class="card">
+              <p><i class="fas fa-tree f-card"></i></p>
+              <h3>26</h3>
+              <p class="cc-1">Outdoor Lanes</p>
+            </div>
           </div>
-        </div>
-        <div class="counter-column">
-          <div class="card">
-            <p><i class="fas fa-door-closed"></i></p>
-            <h3>2</h3>
-            <p>Private Rooms</p>
+          <div class="counter-column">
+            <div class="card">
+              <p><i class="fas fa-door-closed f-card"></i></p>
+              <h3>2</h3>
+              <p class="cc-1">Private Rooms</p>
+            </div>
           </div>
-        </div>
-        <div class="counter-column">
-          <div class="card">
-            <p><i class="far fa-smile"></i></p>
-            <h3>100+</h3>
-            <p>Happy Customers</p>
+          <div class="counter-column">
+            <div class="card">
+              <p><i class="far fa-smile f-card"></i></p>
+              <h3>100+</h3>
+              <p class="cc-1">Happy Customers</p>
+            </div>
           </div>
         </div>
       </div>
       <!-- end counter section -->
       <!-- start map section -->
       <section class="map-section">
```

Comment:
There are some validation errors between lines 131 and 165 <section> element. It looks like there are some divs that have no closing tag. It can be difficult to keep track of so many elements when they become this nested. I might suggest trying to simplify some of the div containers in the future.  I left a few comments below, you don't need to redo everything, they are just things to keep in mind moving forward.